src/components/grid-aware/HomePageLargeParagraph/HomePageParagraph.module.css

```diff
@@ -0,0 +1,34 @@
+.text {
+  font: var(--font-headline);
+  text-align: center;
+}
+
+.bleedWrapper {
+  display: grid;
+  align-items: center;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+  padding-top: 120px;
+  padding-bottom: 120px;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid areas */
+
+.gridAreaMiddle {
+  grid-column: 2;
```

Comment:
I don't think property is doing anything, since this element's parent does not have `display: grid` set. This probably also means that you can just delete this selector and the component associated with it.

Reply:
👍 Deleted.